projects/speex/build.sh

```diff
@@ -30,11 +30,13 @@ PKG_CONFIG_PATH="$WORK"/lib/pkgconfig ./configure --prefix="$WORK" --enable-stat
 make -j$(nproc)
 make install
 $CXX $CXXFLAGS contrib/oss-fuzz/speexdec_fuzzer.cc -o $OUT/speex_decode_fuzzer_fixed -L"$WORK/lib" -I"$WORK/include" $LIB_FUZZING_ENGINE -lspeex -logg
+echo -en "[libfuzzer]\nmax_len = 4096\n" > $OUT/speex_decode_fuzzer_fixed.options
```

Comment:
Please avoid using `max_len` as per https://github.com/google/oss-fuzz/blob/40dbd8d101b469581e2370aecdb9942939b94385/docs/getting-started/new_project_guide.md#custom-libfuzzer-options-for-clusterfuzz